### PR TITLE
Record the percent of stake held me each nonsigner

### PR DESCRIPTION
**Pull request by mooselumph**
## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by mooselumph** [comment]
Will create a separate PR for setting the start and stop times for the query.

**Review by ian-shim — COMMENTED** [review]
lgtm, I think we can add a test in https://github.com/Layr-Labs/eigenda/blob/master/disperser/dataapi/server_test.go#L316

File: disperser/dataapi/nonsigner_handler.go [hunk 1]
```diff
@@ -85,13 +91,29 @@ func (s *server) getOperatorNonsigningRate(ctx context.Context, intervalSeconds
 				if err != nil {
 					return nil, err
 				}
+
+				opID := [32]byte{}
```
Comment (jianoaix) [inline disperser/dataapi/nonsigner_handler.go:95]: The `op` had already dropped the "0x"

File: disperser/dataapi/nonsigner_handler.go [hunk 2]
```diff
@@ -85,13 +91,29 @@ func (s *server) getOperatorNonsigningRate(ctx context.Context, intervalSeconds
...
+				}
+
 				nonsignerMetric := OperatorNonsigningPercentageMetrics{
 					OperatorId:           fmt.Sprintf("0x%s", op),
 					OperatorAddress:      nonsignerIdToAddress[op],
 					QuorumId:             q,
 					TotalUnsignedBatches: unsignedCount,
 					TotalBatches:         totalCount,
 					Percentage:           pf,
+					StakePercentage:      stakePercentage,
```
Comment (jianoaix) [inline disperser/dataapi/nonsigner_handler.go:116]: If this field is 0, it indicates the operator is no longer in the active set at block endBlock?

Comment (mooselumph) [reply disperser/dataapi/nonsigner_handler.go:114]: Yes, that's the correct interpretation.

**Review by ian-shim — APPROVED** [review]

**Pull request merged**